src/app/modules/components/date-picker-range/date-picker-range.component.ts

```diff
@@ -45,12 +45,17 @@ export class DatePickerRangeComponent implements OnInit {
     end: Moment;
   }>();
   range: FormGroup = new FormGroup({});
-
+  maxDate: Date = new Date();
   subscriptions: Subscription[] = [];
 
   constructor(private route: ActivatedRoute) {}
 
   ngOnInit(): void {
+    const currentYear = this.maxDate.getFullYear();
+    const currentMont = this.maxDate.getMonth();
+    const currentDay = this.maxDate.getDate();
+
+    this.maxDate = new Date(currentYear, currentMont, currentDay);
```

Comment:
Só pra corrigir um errinho besta de digitação
```suggestion
    const currentMonth = this.maxDate.getMonth();
    const currentDay = this.maxDate.getDate();

    this.maxDate = new Date(currentYear, currentMonth, currentDay);
```

Reply:
Atualizei, muito obrigado